setup.sh

```diff
@@ -52,7 +67,16 @@ function _show_local_usage
             Allows container access to the bind mount content that is private and
             unshared with other containers on a SELinux-enabled host.
 
-${ORANGE}EXIT STATUS${RESET}
+    ${LBLUE}Podman${RESET}
+        -R
```

Comment:
No show-stopper. But I was wondering, why you chose an upper-case 'R'. With exception of `-Z`, all other parameters are lower-case.

Reply:
The full story is this: I war originally trying to use `--rootless`, but since `getopts` does not care about double-dashed flags, and I did not want to increase the complexity of the script, I figured I should go with a single-letter-option. When I used `r` -- and I did in the beginning -- I always found myself reminded of `--recursive` and I knew some scripts where an uppercase letter denoted that there is a long version available, so I chose `-R` :)